pkgs/racket-doc/json/json.scrbl

```diff
@@ -74,18 +75,26 @@ the @rfc for more information about JSON.
   the range of @tt{U+10000} and above are encoded as two @tt{\uHHHH}
   escapes, see Section 2.5 of the @|rfc|.
 
+  If @racket[indent] is supplied as an integer, output will be "pretty
+  printed" at the specified indentation level.
+
 @examples[#:eval ev
   (with-output-to-string
     (λ () (write-json #hasheq((waffle . (1 2 3))))))
   (with-output-to-string
     (λ () (write-json #hasheq((와플 . (1 2 3)))
                       #:encode 'all)))
+  (displayln
```

Comment:
If you really want to display, I think you can use `write-json` directly without `displayln` and `with-output-to-string`. For consistency though, I think it probably should evaluate to a string value.